sample/redo/lib/main.dart

```diff
@@ -238,7 +253,12 @@ class _MyHomePageState extends State<MyHomePage> {
   Future<void> _handleDone(int id, bool complete) =>
       _updateTodo({'id': id, 'complete': complete});
 
-  Future<void> _handleRemove(int id) => _deleteTodo({'id': id});
+  Future<void> _handleRemove(int id) {
+    _deleteTodo({'id': id});
```

Comment:
[fix?] async/await

Reply:
Not needed, but I'll change the return type to just `void`

Reply:
done